### PR TITLE
Fix a few includes

### PR DESCRIPTION
**Pull request by roversch**
This PR is parallel to PR https://github.com/giaf/blasfeo/pull/48 in blasfeo. 

- Fix a few includes to get rid of warnings in acados.
- Small change to compile HPIPM for dSpace

### CODE REVIEW

File: ocp_qp/d_ocp_qp_kkt.c [hunk 1]
```diff
@@ -32,6 +32,7 @@
 #include <blasfeo_target.h>
 #include <blasfeo_common.h>
 #include <blasfeo_d_aux.h>
+#include <blasfeo_d_aux_ext_dep.h>
```
Comment (giaf) [inline ocp_qp/d_ocp_qp_kkt.c]: why is this needed? there should be no printings of blasfeo stuff (otherwise please report here the issue)

Comment (roversch) [reply ocp_qp/d_ocp_qp_kkt.c]: You're right, this is not necessary. I'll remove it
